ecosystem/theme-default/src/client/utils/updateDeviceStatus.ts

```diff
@@ -0,0 +1,30 @@
+import { onMounted } from 'vue'
+import cssVars from '../styles/_variables.module.scss?module'
+
+export enum DeviceType {
+  MOBILE = 'mobile',
+}
+
+const DeviceTypeMap = {
+  [DeviceType.MOBILE]: Number.parseInt(cssVars.mobile?.replace('px', ''), 10),
+}
+
+/**
+ * add listener to detect screen though device type
+ */
+export const updateDeviceStatus = (
```

Comment:
This method is more like a composable than a util. It's required to be used in component setup. So better to move it under composables dir and rename it with `use` prefix

Reply:
I added it under composites